Makefile

```diff
@@ -13,7 +13,7 @@ version+=master
 
 all: dev-setup build-production
 
-dev-setup: clean-dev composer-install-dev npm-init
+dev-setup: clean-dev composer-install-production npm-init
```

Comment:
@nickvergessen I think this was a test that you did. Isn't the correct -dev?

Reply:
No, this was basically the main reason. Default instructions otherwise break the Nextcloud instance